tus-clients/python/requirements.txt

```diff
@@ -0,0 +1,2 @@
+tuspy
```

Comment:
recommend running `pip freeze > requirements.txt` so that all the dependencies come out.